hera_pspec/utils.py

```diff
@@ -12,6 +12,50 @@
 from datetime import datetime
 
 
+
+def sinc_downweight_mat_inv(nchan, df, weights, filter_centers, filter_widths, filter_factors):
+    """
+    Computes inverse of clean weights for a baseline.
+    This form of weighting is diagonal in delay-space and down-weights tophat regions
+    Parameters
+    ----------
+    nchan: integer
+        Number of channels on baseline
+    df: float
+        channel width (Hz)
+    weights: array-like (complex or float)
```

Comment:
let's specify `shape=(Nfreqs,)`

Reply:
I've added a comment that the shape should be (Nfreqs,)